src/util.ts

```diff
@@ -41,16 +48,16 @@ export function getSentryUrl({
   hint,
   sentryOrg,
   baseSentryUrl,
-  hub,
+  client,
 }: {
-  hint?: EventHint;
+  hint?: EventHint | EventHintV8;
   sentryOrg: string;
   baseSentryUrl: string;
-  hub: Hub;
+  client?: Client | ClientV8;
```

Comment:
switched this to `client` since so it's easier to consume by both integrations

Reply:
This would be a breaking change.

Reply:
Isn't this an internal function? Pre-v8 will still get the client via the `hub`. It worked when I tested it with Sentry 7 anyway 🤷‍♂️